includes/wp-cli/class-one-time-fixers.php

```diff
@@ -1,179 +0,0 @@
-<?php
-/**
- * Clean up plugin data via WP-CLI
- *
- * @package a8c_Cron_Control
- */
-
-namespace Automattic\WP\Cron_Control\CLI;
-
-/**
- * Run one-time fixers for Cron Control
- */
-class One_Time_Fixers extends \WP_CLI_Command {
-	/**
-	 * Remove all data stored in the plugin's table
-	 *
-	 * @subcommand remove-all-plugin-data
-	 * @synopsis [--dry-run=<dry-run>]
-	 * @param array $args Array of positional arguments.
-	 * @param array $assoc_args Array of flags.
-	 */
-	public function purge( $args, $assoc_args ) {
-		global $wpdb;
-
-		// Are we actually destroying any data?
-		$dry_run = true;
-
-		if ( isset( $assoc_args['dry-run'] ) && 'false' === $assoc_args['dry-run'] ) {
-			$dry_run = false;
-		}
-
-		// Provide some idea of what's going on.
-		\WP_CLI::log( __( 'CRON CONTROL', 'automattic-cron-control' ) . "\n" );
-
-		$table_name = \Automattic\WP\Cron_Control\Events_Store::instance()->get_table_name();
-		$count = (int) $wpdb->get_var( "SELECT COUNT(ID) FROM {$table_name}" ); // Cannot prepare table name. @codingStandardsIgnoreLine
-
-		if ( $count > 1 ) {
-			/* translators: 1: Event count */
-			\WP_CLI::log( sprintf( __( 'Found %s total items', 'automattic-cron-control' ), number_format_i18n( $count ) ) . "\n\n" );
-
-			if ( $dry_run ) {
-				\WP_CLI::error( __( 'Stopping as this is a dry run!', 'automattic-cron-control' ) );
-			}
-		} else {
-			\WP_CLI::error( __( 'No entries found...aborting!', 'automattic-cron-control' ) );
-		}
-
-		// Should we really destroy all this data?
-		if ( ! $dry_run ) {
-			\WP_CLI::log( __( 'This process will remove all data for the Cron Control plugin', 'automattic-cron-control' ) );
-			\WP_CLI::confirm( __( 'Proceed?', 'automattic-cron-control' ) );
-			\WP_CLI::log( "\n" . __( 'Starting...', 'automattic-cron-control' ) . "\n" );
-		}
-
-		// Don't create new events while deleting events.
-		\Automattic\WP\Cron_Control\_suspend_event_creation();
-
-		// Don't truncate as it requires DROP and resets auto-increment value.
-		if ( ! $dry_run ) {
-			$wpdb->query( "DELETE FROM {$table_name}" ); // Cannot prepare table name. @codingStandardsIgnoreLine
```

Comment:
Started by moving this to a new store method, but ultimately decided it's not really something we need. If somebody is embarking w/ a goal like this, they'll have to know what they are doing and can pretty much just as easily run the delete SQL manually. At least just as easily as one could track down this CLI, if not more likely :)

And similar to the cache CLI, rather than log a CLI error, can just delete the whole CLI pretty safely.